src/MoviesContext.js

```diff
@@ -0,0 +1,85 @@
+import React, {
```

Comment:
please move this file to `src/contexts/MoviesContext.jsx`

Reply:
Done